charts/spark-operator-chart/templates/deployment.yaml

```diff
@@ -108,6 +108,15 @@ spec:
         - -leader-election-lock-namespace={{ default .Release.Namespace .Values.leaderElection.lockNamespace }}
         - -leader-election-lock-name={{ .Values.leaderElection.lockName }}
         {{- end }}
+        {{- if .Values.webhook.enable }}
+        readinessProbe:
+          httpGet:
+            scheme: HTTPS
+            path: /readyz
+            port: {{ .Values.webhook.port }}
+          initialDelaySeconds: 5
+          periodSeconds: 10
```

Comment:
I thought these were sensible default values but happy for comments

Reply:
Let's make this configurable from the values file in case it doesn't work for some environments.

Reply:
Added configurability and a quick helm unittest as well